internal/fsutil/create_test.go

```diff
@@ -184,17 +194,23 @@ func (s *S) TestCreate(c *C) {
 		c.Assert(err, IsNil)
 		c.Assert(testutil.TreeDump(dir), DeepEquals, test.result)
 
-		if entry.Link != "" && entry.Mode&fs.ModeSymlink == 0 {
-			// Entry is a hardlink.
+		isHardLink := entry.Link != "" && entry.Mode&fs.ModeSymlink == 0
+		// Special case for hard links: use the same file info to compare the
+		// path and the link.
```

Comment:
I think this comment is describing what you can already see in the code below so it is better to remove it.

Reply:
Fixed in [4090b4c](https://github.com/letFunny/chisel/pull/17/commits/4090b4c4c27eda1bd386bb1757572a4529887ad9).